packages/fluentui/react-icons-northstar/test/components/components-test.tsx

```diff
@@ -0,0 +1,24 @@
+import * as icons from '@fluentui/react-icons-northstar';
+import * as fs from 'fs';
+import * as React from 'react';
+
+const iconsDir = 'src/components';
```

Comment:
```suggestion
const iconsDir = path.resolve(__dirname, 'src/components');
```

Reply:
But the `src/components` is not in the current `test` folder. And would like to avoid navigating up the parent to the folder.. Let me know if you have some other idea..